src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -153,10 +152,21 @@ object DatabaseConversions extends LazyLogging {
 
         val untouchedAccounts =
           inactiveBakers
-            .map(_.copy(isActiveBaker = Some(false)))
-            .filterNot(accountsHistoryRow => touched.contains(accountsHistoryRow.accountId))
+            .filterNot(inactiveAccountsRow => touched.contains(inactiveAccountsRow.accountId))
+            .map {
+              _.into[Tables.AccountsHistoryRow]
+                .withFieldConst(
+                  _.asof,
+                  Timestamp.from(blockTaggedAccounts.timestamp.getOrElse(Instant.ofEpochMilli(0)))
+                )
+                .withFieldConst(_.cycle, blockTaggedAccounts.cycle)
+                .withFieldConst(_.blockLevel, blockTaggedAccounts.blockLevel)
+                .withFieldConst(_.blockId, blockTaggedAccounts.blockHash.value)
```

Comment:
Minor: might it read better if these fields are bound via pattern matching at line 133?

Reply:
I would agree in a smaller method, but I think it is clearer this way in the bigger method